cosmosdb/gremlin/ps-gremlin-ru-get.ps1

```diff
@@ -5,7 +5,7 @@
 # --------------------------------------------------
 # Variables - ***** SUBSTITUTE YOUR VALUES *****
 $resourceGroupName = "myResourceGroup" # Resource Group must already exist
-$accountName = "myaccount" # Must be all lower case
+$accountName = "myAccount" # Must be all lower case
```

Comment:
Why are you changing the placeholder for the account name to be mixed case when it needs to be lower case? This may cause confusion with customers.

Reply:
Typo - fixed in subsequent PR. Thanks for catching!